### PR TITLE
[JARS-88] Start one process for celery beat

### PR DESCRIPTION
**Pull request by hashknot**
Start celery beat in a separate independent
process.

### CODE REVIEW

**Review by prnvshrn — APPROVED** [review]

File: files/celery_beat_start [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+NAME="amphora"                                  # Name of the application
+DJANGODIR=/var/opt/wsgi/amphora/amphora             # Django project directory
+SOCKFILE=/var/opt/wsgi/amphora/run/amphora.sock  # we will communicte using this unix socket
+USER=amphora                                        # the user to run as
+GROUP=webapps                                     # the group to run as
+NUM_WORKERS=3                                     # how many worker processes should Gunicorn spawn
+DJANGO_SETTINGS_MODULE=jars.settings             # which settings file should Django use
+DJANGO_WSGI_MODULE=jars.wsgi                     # WSGI module name
+DJANGO_CELERY_MODULE=jars.celery                     # WSGI module name
```
Comment (jdamerow) [inline files/celery_beat_start]: let's remove the ones that are not used

Comment (jdamerow) [reply files/celery_beat_start]: and let's adjust the NAME variable to accurately reflect which process is started

**Review by jdamerow — CHANGES_REQUESTED** [review]
just some small changes

**Comment by hashknot** [comment]
- Removed unnecessary variable definitions
- Updated NAME variable definition

**Review by jdamerow — APPROVED** [review]

**Pull request merged**